vehicle-tracker/vehicle_tracker_app/lib/features/inbox/widgets/inbox_header_widget.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:digit_components/digit_components.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:vehicle_tracker_app/features/inbox/controllers/header_controllers.dart';
+
+class InboxHeaderWidget extends StatelessWidget {
+  const InboxHeaderWidget({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    HeaderControllers headerControllers = Get.find<HeaderControllers>();
+
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 10.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          const Text("Home / FSM / Inbox"),
+          const Text("INBOX", style: TextStyle(fontSize: 30, fontWeight: FontWeight.bold)),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
```

Comment:
create a map and iterate through map

Reply:
Can you explain a little bit more for this